pkg/controller/model/grafanaDeployment.go

```diff
@@ -73,7 +73,10 @@ func getAffinities(cr *v1alpha1.Grafana) *v13.Affinity {
 }
 
 func getSecurityContext(cr *v1alpha1.Grafana) *v13.PodSecurityContext {
-	var securityContext = v13.PodSecurityContext{}
+	var fsGroup int64 = 472 //grafana GID in docker image
```

Comment:
Is this always 472? When deploying on OpenShift 4.6 i get the following error:

```
pods "grafana-deployment-6c4b94f695-" is forbidden: unable to validate
        against any security context constraint: [provider restricted:
        .spec.securityContext.fsGroup: Invalid value: []int64{472}: 472 is not
        an allowed group]
```

Reply:
Just got a chance to look at this. I don't have any experience with OpenShift, so I did a bit of digging and it looks like OpenShift handles `securityContext` a lot different than vanilla Kubernetes.

With this change, OpenShift users would have to modify their security context constraints to allow `fsGroup: 472` or override `fsGroup` to something in their project's allowed range (or null, which would let OpenShift set a value automatically). I'm guessing this project would prefer to not introduce breaking changes for OpenShift users.

Looking at other repositories, I've found that they have specific OpenShift examples due to it's differences from upstream Kubernetes.

Given that, maybe its just best to back out the changes in this file and just update the example at `deploy/examples/persistentvolume` with the securityContext and a note for OpenShift users?

Reply:
Yes, that approach sounds good to me. I would prefer the example to work on vanilla Kubernetes, so adding a note for the special case on OpenShift make sense.